src/boxed.tsx

```diff
@@ -13,16 +13,17 @@ type StylesProps = {
 
 const useStyles = createUseStyles(({colors, isDarkMode}) => ({
     boxed: {
-        background: ({isInverseInside}: StylesProps) =>
+        backgroundColor: ({isInverseInside}: StylesProps) =>
```

Comment:
I think this change will break o2-classic, because colors.backgroundBrand is a gradient in o2 classic

Reply:
ok! backgroundColor is not needed any more. I had to specify it in order to set a specific backgroundClip for the transparent border. But as we agreed to remove the border, those changes have been reverted.